README.md

```diff
@@ -49,36 +50,37 @@ npm install react-colorful --save
 ## Usage
 
 ```js
-import ColorPicker from "react-colorful";
+import { HexColorPicker } from "react-colorful";
 import "react-colorful/dist/index.css";
 
 const YourComponent = () => {
   const [color, setColor] = useState("#aabbcc");
-  return <ColorPicker color={color} onChange={setColor} />;
+  return <HexColorPicker color={color} onChange={setColor} />;
```

Comment:
While we're talking about breaking changes, can I get your thoughts on naming that prop `color`?

It's happened a few times now, where I had to stop and think what a `color` property would be on a color picker. I'm wondering if something like `value` might be more idiomatic and easier to understand?

My problem with `color` is that, on a component, my first assumption would be that I'm setting some sort of styling for it. Maybe it's just me, but it takes a second to remember that `color` is the current value of the component. 

I personally think `value` would be more idiomatic, as then it matches more of the environment, for example, input fields. I feel that `value` and `onChange` fits better than `color` and `onChange`. While yes, it makes sense to set the color of a color picker, I think it's more accurate to say we're setting the current value of the color picker which just happens to be some representation of color. 

Just an idea I had.

Reply:
I had the same thought recently. The only thing that I'm afraid of is inconsistency with the color picker.

@molefrog We probably need 3rd mind here)

Reply:
Do you want to change it for all components (the pickers and the input), right?

Reply:
Not necessarily **want** to change it, but I had the thought that it could be more clear and idiomatic if we were to change it. 

But yes, for all the components if we were. 

What's the inconsistency?

Reply:
Forget it) I just thought you want to change `HexColorInput` only.

Reply:
I would keep `color` for now. It's okay I think and I don't want to make the PR bigger.

Reply:
Oh, gotcha. 

Yeah, fair, no problem.